go/enclave/storage/enclavedb/batch.go

```diff
@@ -401,7 +401,7 @@ func ReadReceipt(db *sql.DB, hash common.L2TxHash, config *params.ChainConfig) (
 	return receipts[0], nil
 }
 
-func ReadTransaction(db *sql.DB, txHash gethcommon.Hash) (*types.Transaction, gethcommon.Hash, uint64, uint64, error) {
+func ReadTransaction(db *sql.DB, txHash gethcommon.Hash) (*types.Transaction, common.L2BatchHash, uint64, uint64, error) {
```

Comment:
this is an unrelated change. But it confused me, so I used the type alias to clarify the meaning